src/linter/block.go

```diff
@@ -148,6 +149,11 @@ func (b *blockWalker) reportDeadCode(n ir.Node) {
 		return
 	}
 
+	if b.containsDisableInspection(n, "PhpUnreachableStatementInspection") {
+		b.ctx.deadCodeReported = true
+		return
+	}
```

Comment:
I would move this `if` statement below the switch, closer to the actual `Report` call.

Reply:
But it will probably make phpdoc matching harder as you'll have to save that comment somewhere else. Nevermind then.